### PR TITLE
Paths on Gauntlet vs Binary

### PR DESCRIPTION
**Pull request by RodrigoAD**

### CODE REVIEW

File: gauntlet/package.json [hunk 1]
```diff
@@ -35,7 +35,12 @@
       "macos"
     ],
     "outputPath": "bin",
-    "assets": "node_modules/usb/prebuilds"
+    "assets": [
+      "node_modules/usb/prebuilds",
+      "packages/gauntlet-solana-contracts/artifacts",
+      "packages/gauntlet-solana-contracts/networks",
+      "packages/gauntlet-serum-multisig/artifacts"
```
Comment (RodrigoAD) [inline gauntlet/package.json]: Try:
```suggestion
      "packages/gauntlet-solana-contracts/artifacts/**",
      "packages/gauntlet-solana-contracts/networks/**",
      "packages/gauntlet-serum-multisig/artifacts/**"
```

File: gauntlet/packages/gauntlet-solana-contracts/src/cli.ts [hunk 2]
```diff
@@ -7,7 +7,7 @@ import { commands } from '.'
   try {
     const networkPossiblePaths = ['./networks', './packages/gauntlet-solana-contracts/networks']
     const networkPath = networkPossiblePaths.filter((networkPath) =>
-      existsSync(path.join(process.cwd(), networkPath)),
+      existsSync(path.join(__dirname, networkPath)),
```
Comment (RodrigoAD) [inline gauntlet/packages/gauntlet-solana-contracts/src/cli.ts]: This is tricky, cause we need to provide a way to the user if he wants to override the networks env config. Maybe they could have the setup like:
```
./
   gauntlet-macos
   networks/
```
On that case, we'd want that to work, and in that case we should use `process.cwd`

**Comment by kylesmartin** [comment]
@RodrigoAD I made some updates here. With these changes, the binary functions anywhere and allows you to override environment variables with /networks or .env

**Review by kylesmartin — APPROVED** [review]
Looks good!

**Pull request merged**